cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -643,62 +356,72 @@ SwiftGenerator::writeDocSentence(IceInternal::Output& out, const StringList& lin
 void
 SwiftGenerator::writeDocSummary(IceInternal::Output& out, const ContainedPtr& p)
 {
-    DocElements doc = parseComment(p);
-
-    string n = fixIdent(p->name());
+    CommentPtr doc = p->parseComment(swiftLinkFormatter, true, true);
+    if (!doc)
+    {
+        return;
+    }
```

Comment:
The only difference is that the `parseComment` in the parser can return `nullptr` if there is no comment, whereas the old `slice2swift` specific `parseComment` would always return a `DocElements`.

So, now we do need to check if `nullptr` was returned before generating anything.